src/sensors/can/motor.hpp

```diff
@@ -7,7 +7,7 @@
 
 /// @brief Defines the motor controller types. Enum values chosen based on the yaml config specification
 enum MotorControllerType {
```

Comment:
enum class would make it so you don't have to worry about the implicit type, and check at compile time any comparisons are valid.